fermilibpluginpyscf/_run_pyscf.py

```diff
@@ -0,0 +1,255 @@
+    two_electron_integrals = numpy.empty((n_orbitals, n_orbitals,
+                                          n_orbitals, n_orbitals))
+
+    # Unpack symmetry.
+    n_pairs = n_orbitals * (n_orbitals + 1) // 2
+    if two_electron_compressed.ndim == 2:
+
+        # Case of 4-fold symmetry.
+        assert(two_electron_compressed.size == n_pairs ** 2)
+        pq = 0
+        for p in range(n_orbitals):
+            for q in range(p + 1):
+                rs = 0
+                for r in range(n_orbitals):
+                    for s in range(r + 1):
+                        pqrs_value = two_electron_compressed[pq, rs]
+                        two_electron_integrals[p, s, r, q] = float(pqrs_value)
+                        two_electron_integrals[q, s, r, p] = float(pqrs_value)
+                        two_electron_integrals[p, r, s, q] = float(pqrs_value)
+                        two_electron_integrals[q, r, s, p] = float(pqrs_value)
+                        rs += 1
+                pq += 1
+    else:
+
+        # Case of 8-fold symmetry.
+        assert(two_electron_compressed.size == n_pairs * (n_pairs + 1) // 2)
+        pq = 0
+        pqrs = 0
+        for p in range(n_orbitals):
+            for q in range(p + 1):
+                rs = 0
+                for r in range(p + 1):
+                    for s in range(r + 1):
+                        if pq >= rs:
+                            pqrs_value = two_electron_compressed[pqrs]
+                            two_electron_integrals[p, s,
+                                                   r, q] = float(pqrs_value)
+                            two_electron_integrals[q, s,
+                                                   r, p] = float(pqrs_value)
+                            two_electron_integrals[p, r,
+                                                   s, q] = float(pqrs_value)
+                            two_electron_integrals[q, r,
+                                                   s, p] = float(pqrs_value)
+                            two_electron_integrals[s, p,
+                                                   q, r] = float(pqrs_value)
+                            two_electron_integrals[s, q,
+                                                   p, r] = float(pqrs_value)
+                            two_electron_integrals[r, p,
+                                                   q, s] = float(pqrs_value)
+                            two_electron_integrals[r, q,
+                                                   p, s] = float(pqrs_value)
+                            pqrs += 1
+                        rs += 1
+                pq += 1
+
+    # Return.
+    return one_electron_integrals, two_electron_integrals
+
+
+def run_pyscf(molecule,
+              run_scf=True,
+              run_mp2=False,
+              run_cisd=False,
+              run_ccsd=False,
+              run_fci=False,
+              verbose=False):
+    """This function runs a Psi4 calculation.
```

Comment:
I fixed the missing run_mp2 option. But I did not add the space if that is what you are asking for. I want all files in FermiLib to be consistent. If you insist on this space then please update all others files. We can do that before moving out of alpha but I don't want to be half in one convention and half in another.